### DISCUSSION TITLE
Sorting included relationships

### DISCUSSION THREAD

**Discussion by glonaldo70** [tailflow/laravel-orion#285] [Ideas]
Something like this
....ebove filtrer,
  "includes": [
    {
      "relation": "messages",
      "sort": [
        {
          "field": "created_at",
          "direction": "desc"
        }
      ]
    }
  ]

i want to get all message of user sorted. i can not go through message controller directly, i have to pass to user before

**Comment by alexzarbn** [comment]
Hi @glonaldo70,

It is currently not possible to sort included relationships. I'm marking the issue as "enhancement" for later implementation.
